heroku/cli
# refactor: migrate pg:copy to oclif/core

[Work item](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bCpAVYA0/view)

## Notes
I took the opportunity here to rename and move a couple of utility functions, so you'll see a lot of files touched in this PR, but it's just a function name and path change.
- We have been using the `confirmApp` function from the lib/apps directory in a lot of non-apps commands, so I renamed it `confirmCommand` and moved it to the root of the lib directory.
- There was some confusion around calling the command `attachment` in the lib/pg/fetcher.ts file. I renamed it `getAttachment` and updated all the places it gets called for clarity.

## Testing
- Pull down this branch and run `yarn install` and `yarn build`
- Find or set up an app with two databases
- run `./bin/run pg:copy --help` just to check
- run `./bin/run pg:copy -a APP_NAME SOURCE_DB_NAME TARGET_DB_NAME`
    - If one or both of the databases are postgres, you should get a message asking for confirmation and then the operation should succeed
    - If neither of the databases are postgres, you should get an error message
